examples/plugins/ethdoc/src/ethdoc.ts

```diff
@@ -0,0 +1,115 @@
+import {
+  CompilationResult,
+  CompilatedContract,
+  DeveloperDocumentation,
```

Comment:
All these types are not really related to `remix-plugin` , we could easily have an another NPM package for referencing all the solidity related types ( I recall that something we already being discussed by the solidity team right?)

Reply:
We were thinking about putting them into the solc-js repository. But I'll need to change several things to their repo before. So it might take more time.